.pre-commit-config.yaml

```diff
@@ -30,3 +30,13 @@ repos:
     rev: "0.76"
     hooks:
       - id: flynt
+  - repo: https://github.com/pycqa/isort
+    rev: 5.10.1
+    hooks:
+      - id: isort
+        args: ["--profile", "black", "--filter-files"]
```

Comment:
Thoughts on adding an isort config file instead of specifying this here?

Reply:
I could go either way. Do you have a strong preference? I suppose the top level configuration file makes it easier to run `isort` yourself, although I tend to dislike the proliferation of tons of top level dotfiles.

Reply:
I know VSCode has an isort integration, I presume it'd respect an isort.cfg.

Reply:
Isort supports pyproject.toml for config, so maybe we should use that to avoid giving isort its own config?